.github/workflows/ci-e2e-kafka.yml

```
@@ -17,8 +17,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jaeger-version: [v1, v2]  # Adjust if there are specific versions of Jaeger
-    name: kafka ${{ matrix.jaeger-version }}
+        jaeger-version: [v1, v2]
+        version:
+          - major: 3.x
+            distribution: kafka
+          - major: 2.x
+            distribution: kafka
```

```suggestion
        kafka-version: ["3.x", "2.x"]
```
the "distribution" is unnecessary, the whole file is about kafka